build/wix/build32.bat

```diff
@@ -0,0 +1,39 @@
+@echo off
```

Comment:
Yes, I can do that.
I made it that way to be a temporary hack to test package building. My goal is to integrate package building into scons, like this is done for NSIS currently, in another PR.
Scons integration will let these batch files unneeded so I will remove them.